eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -440,7 +440,7 @@
                     viewType: 'asset_image',
                     noLayout: true,
                     params: {
-                        parameters: parameters|default({'alias': 'original'})
+                        parameters: parameters|default({'alias': 'original'})|merge({ alternativeText: field.value.alternativeText })
```

Comment:
Not a Twig expert, but I see a little bit inconsistency - assuming `alternativeText` is key literal, it would look better if surrounded with single quotes as in case of `'alias'`. Can it work like that? :)